src/components/elements/image/index.tsx

```diff
@@ -3,11 +3,13 @@ import ExportedImage from 'next-image-export-optimizer';
 
 import { StyledImageWrapper } from './image.styles';
 
-export type ImageProps = ExportedImageProps;
+export type ImageProps = ExportedImageProps & {
+  $forRichText?: boolean;
+};
 
-export const Image = ({ ...props }: ImageProps) => {
+export const Image = ({ $forRichText, ...props }: ImageProps) => {
   return (
-    <StyledImageWrapper>
+    <StyledImageWrapper $forRichText={$forRichText}>
```

Comment:
I prefer to leave the Image component without any custom props and instead Style it where needed. In this case, in `rich-text/components` WDYT?

``` ts
export const RichTextImage = styled(Image)`
   ...
`
```

You will need to add the className only to be able to style in that way:

```tsx
export const Image = ({ className, ...props }: ImageProps) => {
  return (
    <StyledImageWrapper className={className}>
```

Reply:
Yep, but this would need some styled component instead of the className because of specificity; in this case, it is not bad because we will need to use it !important anyways, but on standard components, it would lead to over use of !important